templates/re-base/finishInstall.js

```diff
@@ -48,28 +52,17 @@ function installDevDependencies() {
     unlinkSync(devDependenciesJsonPath);
 }
 
-// Add `react-native-web` plugin to `.babelrc` file
-function updateBabelrc() {
-  const file = './.babelrc'
-  const babelrc = JSON.parse(readFileSync(file))
-
-  let {plugins} = babelrc
-  if(!plugins) plugins = []
-
-  plugins.push('react-native-web')
-
-  babelrc.plugins = plugins
-
-  writeFileSync(file, JSON.stringify(babelrc, null, 2)+'\n')
-}
-
 function enableWindows() {
     console.log('Enable Windows support to the project...');
     execSync(`react-native windows`);
 
     unlinkSync(resolve('App.windows.js'))
 }
 
+function enableBabelrc() {
+  execSync(install);
+}
```

Comment:
This is not needed, since this script is executed only when creating the project, but after that, the important part is on `install`, that the `react-scripts` package need to be updated, and the `install.js` will be executed anyway, so this is superfluous.